### PR TITLE
Refactor: remove outdated font tag

### PR DESCRIPTION
**Pull request by stu01509**
#### Short description of what this resolves:
According to this PR https://github.com/openemr/openemr/pull/3596#issuecomment-640153784. I found the SQL upgrade result contain outdated font tag, so I created this PR to fix it.

### CODE REVIEW

File: ippf_upgrade.php [hunk 1]
```diff
@@ -163,7 +163,7 @@ function do_visit_form($irow, $encounter, $first)
         }
     }
 
-    echo "<p><font color='green'>Done. Inserted $insert_count visit forms.</font></p>\n";
+    echo "<p class='text-success'>Done. Inserted $insert_count visit forms.</p>\n";
```
Comment (bradymiller) [inline ippf_upgrade.php:166]: Is this change bs4 specific? If so, this script doesn't include bs4.

**Comment by bradymiller** [comment]
hi @stu01509 , Code looks good except for the change in ippf_upgrade.php since there is not bs4 there.

File: ippf_upgrade.php (hunk 1 above)
Comment (tywrenn) [reply ippf_upgrade.php:166]: @bradymiller @stu01509 Maybe it's time to convert this file to BS4 *hint* *hint* *nudge* *nudge*

**Comment by stu01509** [comment]
Hello @bradymiller and @tywrenn 

I created PR #3630, In this PR, I added the Bootstrap.
Please take a look.

**Comment by bradymiller** [comment]
hi @stu01509 , This looks good and bringing into codebase. Thanks for the improvements! -brady

**Pull request merged**